bin/hab-studio

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Copyright:: Copyright 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+function usage() {
+  cat /dev/stdin <<DOC
+Usage: ${0##*/} [SUBCOMMAND]
+
+Utility to configure aspects of your Habitat Studio prior to launch.
+
+SUBCOMMANDS:
+    cleanup                               Remove any state created by any of the ${0##*/} commands.
+    configure-github-account ACCOUNT      Configure studios with GitHub credentials for ACCOUNT.
+DOC
+}
+
+function configure_github_account_usage() {
+  cat /dev/stdin <<DOC
+Usage: ${0##*/} configure-github-account ACCOUNT
+
+Configure studios with GitHub credentials for ACCOUNT.
+
+Requires you have the following environment variables set: 
+    * <ACCOUNT>_GITHUB_TOKEN
+
+When executed, it will do the following:
+    * Add ~/.netrc entry for ACCOUNT in Studio.
+    * Make GITHUB_TOKEN environment variable available in Studio.
+DOC
+}
+
+function configure_github_account() {
+  if [[ -z "$1" || "$1" == "-h" || "$1" == "--help" ]];
+  then
+    configure_github_account_usage
+    exit 0
+  fi
+
+  init
+
+  account="$1"
+  account_env="$(echo "${account^^}" | tr '-' '_')"
+  account_token="${account_env}_GITHUB_TOKEN"
+
+  if [[ -z "${!account_token}" ]];
+  then
+    echo "${0##*/}: Unable to find the required environment variable: $account_token."
+    echo "Make sure it is specified."
+    exit 2
+  fi
+
+  # Add credentials to ~/.netrc
+  if ! grep -w "machine github\.com" $HOME/.netrc > /dev/null
+  then
+    echo -e "machine github.com\n  login ${!account_token}" > $HOME/.netrc
```

Comment:
I wonder if we should override this file. Just a thought. 🤔

Reply:
I'll noodle on that.

Reply:
I think the answer here is "yes" - we want to control this file, especially since it's in the studio.